catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -80,6 +86,8 @@
 
   private static List<MetacardTransformer> transformers = null;
```

Comment:
❗️ This one is a problem for multi-threading since it is updated once and accessed multiple times. Its reference should be marked `volatile`

Reply:
It may be preferable to make this a local variable if possible.

Reply:
probably right since it would be set the first time this command is executed with the current set of transformers and never updated after. Doing so will require properly releasing all services acquired from the bundle context after a particular command execution is done.

Reply:
@paouelle I'll create a ticket to address further concurrency issues in this command in the future

Reply:
Why not fix it here? It should be simple enough to fix

Reply:
@paouelle I created a ticket for this here: https://codice.atlassian.net/browse/DDF-4195